vita3k/audio/src/audio.cpp

```diff
@@ -15,21 +15,19 @@
 // with this program; if not, write to the Free Software Foundation, Inc.,
 // 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+#include "Tracy.hpp"
```

Comment:
Is it needed to add #ifdef TRACY_ENABLE here too?

Reply:
The compiler define check I implemented in `module`, `modules`, `renderer` and `shader` are so that in case a third-party application wants to re-use that code from the emulator, the compiler will not complain about not being able to access the Tracy headers. This was heavily inspired by the fact that vita2hos reuses parts of the emulator, but mainly the renderer and the shader recompiler. The Microprofile implementation didn't guard the header includes in any of the emulator code, so I decided to guard the headers in the core components of the emulator so that projects like vita2hos can compile them without having Tracy in their source tree. Since the audio host implementation is specific to Vita3K, I don't think it will be a problem to leave it like that.